teams/account-management/README.md

```diff
@@ -1,5 +1,54 @@
 # Account Management
 
+## Setup your local dev environment
+
+To get started on local development and testing:
+
+1. **Get your `buffer-dev` environment setup**
+  → https://github.com/bufferapp/buffer-dev
+
+2. **Install the latest version of `yarn`**
+  → [Installing Yarn](https://yarnpkg.com/en/docs/install)
+
+3. **Make sure you have node with version <= 9 (Node v10 is not compatible)**
```

Comment:
@hharnisc - Any nodejs restriction?

Reply:
I think as long as we're on the latest LTS version 8 we should be good. (I usually use `nvm` to manage node version: https://github.com/creationix/nvm)

Reply:
You are right, Dan prepared a PR to change the nodejs version of the buffer-login to use the latest active LTS